_data/docs-home.yml

```diff
@@ -59,6 +59,8 @@ toc:
     path: /docs/iot-gateway/sigfox-iot-data-visualization/
   - title: Persist and visualize IBM Watson IoT data
     path: /docs/iot-gateway/integration-with-ibm-watson-iot/
+  - title: Integrate with Telegram Bot
```

Comment:
I would rename this to: 
"Notifications and Alarms on your smartphone using Telegram Bot"

@ashvayka any comments?